src/inmanta/export.py

```diff
@@ -398,7 +398,7 @@ def run(
         resources = self.resources_to_list()
 
         export_done = time.time()
-        LOGGER.debug("Exporter took %0.03f seconds", export_done - start)
+        LOGGER.debug("Serializing resources took %0.03f seconds", export_done - start)
```

Comment:
Yes. 

How would you report the timings?  (I'm trying to produce understandable output, but not too fine grained)

Reply:
I honestly don't know. I'm just slightly worried that this will present an inaccurate picture, which is what we're trying to solve with this PR in the first place.

Reply:
Some suggestions that use our lingo and not per se that of distributed systems

- "Generating resources from the compiled model"
- "Generating desired state from the compiled model"
- ...